src/main/java/com/acoustic/salarycalculator/calculator/SalaryCalculator.java

```diff
@@ -1,17 +1,17 @@
-package org.example.enumApproch.enums;
+package com.acoustic.salarycalculator.calculator;
 
```

Comment:
in general lambdas are good as a one linear, if there is more logic you could extract those to separate methods like  NET_YEARLY(gross -> NET.getOperator()
            .apply(gross)
            .multiply(BigDecimal.valueOf(Rates.MONTH_NUMBER.getRate()))
            .setScale(2, RoundingMode.HALF_EVEN), "Yearly net amount: "),
to :
    NET_YEARLY(getNetYearly(), "Yearly net amount: "),
and
    private static UnaryOperator<BigDecimal> getNetYearly() {
        return gross -> NET.getOperator()
                .apply(gross)
                .multiply(BigDecimal.valueOf(Rates.MONTH_NUMBER.getRate()))
                .setScale(2, RoundingMode.HALF_EVEN);
    }

Reply:
fixed

Reply:
method getTaxAmount contains 2 almost the same blocks, this could be extracted to sth like: 
private static UnaryOperator<BigDecimal> getTaxAmount() {
        return gross -> (GROSS_YEARLY.getOperator()
                .apply(gross)
                .compareTo(BigDecimal.valueOf(Rates.TAX_GROSS_AMOUNT_TRASHOLD.getRate())) < 0) ?
                getTaxAmountBasedOnRate(gross, Rates.TAX_RATE_17) : getTaxAmountBasedOnRate(gross, Rates.TAX_RATE_32);
    }    
    

    private static BigDecimal getTaxAmountBasedOnRate(BigDecimal gross, Rates rate) {
        return gross.subtract(TOTAL_ZUS.getOperator().apply(gross))
                .subtract(HEALTH.getOperator().apply(gross))
                .multiply(BigDecimal.valueOf(rate.getRate()))
                .setScale(2, RoundingMode.HALF_EVEN);
    }